src/update-manager.ts

```diff
@@ -769,7 +758,11 @@ export default class UpdateManager {
         (this.store.fetcher.load(doc) as Promise<Response>).then(response => {
           this.update(deletions, insertions, callback, true)
         }, err => {
-          throw new Error(`Update: Can't read ${doc} before patching: ${err}`)
+            if (err.response.status === 404) { // nonexistent files are fine
+              this.update(deletions, insertions, callback, true)
```

Comment:
Maybe create a test to cover this branch?

Reply:
Done